### ISSUE TITLE
[HELP] Failing with ruby 2.6 - KeyError (key not found)

### ISSUE THREAD

**Issue by djberg96** [mvz/ripper_ruby_parser#62]
Ruby 2.6.2 on Mojave (full env below if needed).

Just for kicks, tried to run our setup script for ManageIQ using ruby 2.6.2. It's bombing with:

`KeyError (key not found: s(:@period, ".", s(6, 19)))`

This is coming from a bit of code that does this:

`parsed = RipperRubyParser::Parser.new.parse(content, filename)`

Where `filename` is at https://github.com/ManageIQ/manageiq/blob/master/app/models/ext_management_system.rb, and the `content` is an `IO.read` of that file.

It works fine with Ruby 2.4.x, but bombs with Ruby 2.6.x. Any ideas?

Full environment:

```
>gem env
RubyGems Environment:
  - RUBYGEMS VERSION: 3.0.3
  - RUBY VERSION: 2.6.2 (2019-03-13 patchlevel 47) [x86_64-darwin18]
  - INSTALLATION DIRECTORY: /Users/dberger/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0
  - USER INSTALLATION DIRECTORY: /Users/dberger/.gem/ruby/2.6.0
  - RUBY EXECUTABLE: /Users/dberger/.rbenv/versions/2.6.2/bin/ruby
  - GIT EXECUTABLE: /usr/local/bin/git
  - EXECUTABLE DIRECTORY: /Users/dberger/.rbenv/versions/2.6.2/bin
  - SPEC CACHE DIRECTORY: /Users/dberger/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Users/dberger/.rbenv/versions/2.6.2/etc
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-darwin-18
  - GEM PATHS:
     - /Users/dberger/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0
     - /Users/dberger/.gem/ruby/2.6.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
     - :sources => ["https://rubygems.org/"]
     - "gem" => "--document=yri"
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /Users/dberger/.rbenv/versions/2.6.2/bin
     - /usr/local/Cellar/rbenv/1.1.1/libexec
     - /usr/local/opt/icu4c/sbin
     - /usr/local/opt/icu4c/bin
     - /usr/local/opt/postgresql@9.5/bin
     - /Users/dberger/.rbenv/shims
     - /usr/local/bin
     - /usr/bin
     - /bin
     - /usr/sbin
     - /sbin
```

**Comment by mvz** [comment]
Hi @djberg96, Ruby 2.6 is indeed not yet supported in the released version, but I'll release a new version with support for 2.6 soon.

**Comment by mvz** [comment]
Version 1.5.0 has been released that supports Ruby 2.6.
